js/app.js

```diff
@@ -8,16 +8,45 @@ function Post (obj){
   this.image = obj.image;
 }
 
+Post.all = [];
+
 Post.prototype.toHtml = function() {
   var $source = $('#project-template').html();
   var template = Handlebars.compile($source);
+
   return template(this);
 };
 
-portfolioData.forEach(function(ele) {
-  posts.push(new Post(ele));
-});
+Post.loadAll = function(dataPassedIn) {
+  dataPassedIn.forEach(function(ele) {
+    Post.all.push(new Post(ele));
+  });
+};
+
+Post.fetchAll = function (){
+  if (localStorage.projects){
+    $.ajax({
+      type: 'HEAD',
+      url: '../data/projects.json'
+    }).done(function(xhr, message, data) {
```

Comment:
I don't think .done() is the method you want here, mostly because it doesn't seem to have an xhr object associated with it. Try sending a success: function() on the $.ajax object itself. Right now I can't see any of your projects because I get an error.